alpine-38.dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM tozd/runit:alpine-38
+
+EXPOSE 25/tcp 465/tcp 587/tcp
+
+VOLUME /var/log/postfix
+VOLUME /var/spool/postfix
+
+ENV MAILNAME mail.example.com
+ENV MY_NETWORKS 172.17.0.0/16 127.0.0.0/8
+ENV MY_DESTINATION localhost.localdomain, localhost
+ENV ROOT_ALIAS admin@example.com
+
+RUN apk update && apk add postfix rsyslog sed
```

Comment:
Please keep the order and style of files. Then it is easier to compare them.

Reply:
Hm, frankly, those images are build automatically and often from scratch because the base image changed. So this optimization happens only if you are changing this manually, which is rare and probably only during development.

Anyway, `/etc/aliases` has to be available before postfix installation.

And please put every command into its own line.

Also, you can then modify the other dockerfile if you really want to use this order, but make them be the same.

Reply:
Made changes to 1. keep each command on its own line. 2. move /etc/aliases before package installations. 3. keep both the files in similar manner in terms of sequence of execution